pkg/sdk/connector/controlplane/controlplane.go

```diff
@@ -150,15 +153,20 @@ func (cp *ControlPlane) Register(ctx context.Context, integration Integration) e
 		case <-ctx.Done():
 			cp.logger.Debug("Controlplane cancelled via context. Unregistering...")
 			wg.Wait()
+			cp.logger.Debug("wait for all event handlers to finish")
```

Comment:
I think this could even be a Info log ? 
Also, do you think we should add a timeout here? (in case the business logic of the service is stucked and actually never finishes) ?

Reply:
maybe also all other logs here should be info as well?